upgrade-notes/backend_20260109_174841.md

```
@@ -0,0 +1,15 @@
+#### SF product detail - add parameter value color preview ([#4382](https://github.com/shopsys/shopsys/pull/4382))
+
+- [features moved](#movement-of-features-from-project-base-to-packages) from project-base to the framework package:
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)